### DISCUSSION TITLE
getPromptStateless received 403 error when using `LangfuseWeb` SDK

### DISCUSSION THREAD

**Discussion by huangbaitu123** [langfuse/langfuse#3977] [Support] [answered]
```js
import { LangfuseWeb } from "langfuse"

const langfuseweb = new LangfuseWeb({
  publicKey: process.env.LANGFUSE_PUBLIC_KEY,
  baseUrl: process.env.LANGFUSE_BASEURL,
})

langfuseweb.getPromptStateless("ai-tools-dev-test").then((res) => {
  res.data === {
    message: 'Access denied - need to use basic auth with secret key to GET prompts',
    error: 'ForbiddenError'        
  }
}
})
```
LangfuseWeb do not accept secretKey, but getPromptStateless needs basic auth, so this is paradox?

**Comment by marcklingen** [comment, marked as answer]
Thanks for the feedback, `getPromptStateless` is an internal method which is not documented. You need to use `import { Langfuse } from "langfuse"` and authenticate with secret and public key to use `getPrompt()`. See this documentation section for more details: https://langfuse.com/docs/prompts/get-started#use-prompt

I'll create an issue as this is confusing and could be improved, thanks again for the feedback!

**Reply by huangbaitu123** [reply]
Thanks 👍
